src/lib/voiceSearchHelper/__tests__/index-test.ts

```diff
@@ -8,11 +8,14 @@ declare global {
   }
 }
 
+const start = jest.fn();
+const stop = jest.fn();
+
 const createFakeSpeechRecognition = (): jest.Mock => {
   const listeners: any = {};
   const mock = jest.fn().mockImplementation(() => ({
-    start() {},
-    stop() {},
```

Comment:
What was wrong with this?

Reply:
I had to make them a mock function due to [this](https://github.com/algolia/instantsearch.js/pull/3779/files/af65fcd0e9c1ce8122a3e1f39a3aaaa2ef68c97a..ae1f20e93ceffacbd4e67be83b6baaefae858f24#r284716504)